### PR TITLE
[ENH] Describe parametric images above the suffix table, rather than below

### PR DESCRIPTION
**Pull request by CPernet**
relates to https://github.com/bids-standard/bids-specification/issues/1683  information above the table (people do not read what is under - add few common examples to grab attention)

I think the annex table could benefit from having OPTIONAL and MANDATORY for entities, but trough the macro it would break over tables I think (I looked at it and for scared lol)

### CODE REVIEW

**Comment by Remi-Gau** [comment]
python style failure is unrelated: due to the new black release. Will PR to fix.

File: src/modality-specific-files/magnetic-resonance-imaging-data.md [hunk 1]
```diff
@@ -298,7 +298,11 @@ and a guide for using macros can be found at
       ])
 }}
 
-Currently supported parametric maps include:
+Parametric images listed in the table below are typically generated by
+processing a [file collection](../common-principles.md#entity-linked-file-collections).
+Please visit the [file collections appendix](../appendices/file-collections.md) to see the
+list of suffixes available for quantitative MRI (qMRI) applications associated
+with these maps (e.g. [MPM](../glossary.html#mpm-suffixes), [MP2RAGE](../glossary.html#mp2rage-suffixes), [MTR](../glossary.html#mtransfer-entities)). Currently supported parametric maps include:
```
Comment (Remi-Gau) [inline src/modality-specific-files/magnetic-resonance-imaging-data.md:305]:
```suggestion
with these maps
(for example [MPM](../glossary.md#mpm-suffixes),
[MP2RAGE](../glossary.md#mp2rage-suffixes),
[MTR](../glossary.md#mtransfer-entities)).

Currently supported parametric maps include:
```

File: src/modality-specific-files/magnetic-resonance-imaging-data.md [hunk 2]
```diff
@@ -330,11 +339,7 @@ and a guide for using macros can be found at
    )
 }}
 
-Parametric images listed in the table above are typically generated by
-processing a [file collection](../common-principles.md#entity-linked-file-collections).
-Please visit the [file collections appendix](../appendices/file-collections.md) to see the
-list of suffixes available for quantitative MRI (qMRI) applications associated
-with these maps.
+
```
Comment (Remi-Gau) [inline src/modality-specific-files/magnetic-resonance-imaging-data.md:342]:
```suggestion
```

**Review by CPernet — COMMENTED** [review]
thx for the update pal

File: src/modality-specific-files/magnetic-resonance-imaging-data.md [hunk 3]
```diff
@@ -298,6 +298,15 @@ and a guide for using macros can be found at
...
 }}
 
+Parametric images listed in the table below are typically generated by
+processing a [file collection](../common-principles.md#entity-linked-file-collections).
+Please visit the [file collections appendix](../appendices/file-collections.md) to see the
+list of suffixes available for quantitative MRI (qMRI) applications associated
+with these maps
+(for example [MPM](../glossary.md#mpm-suffixes),
+[MP2RAGE](../glossary.md#mp2rage-suffixes),
+[MTR](../glossary.md#mtransfer-entities)).
```
Comment (effigies) [inline src/modality-specific-files/magnetic-resonance-imaging-data.md:308]:
```suggestion
(for example, [MPM](../glossary.md#mpm-suffixes),
[MP2RAGE](../glossary.md#mp2rage-suffixes),
and [MTR](../glossary.md#mtransfer-entities)).
```

**Review by effigies — APPROVED** [review]

**Review by Remi-Gau — APPROVED** [review]

**Pull request merged**